src/main/java/core/basesyntax/service/DataConverter.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.List;
+
+public interface DataConverter {
```

Comment:
Don't begin interface or class implementation with empty lines. Please remove the redundant empty lines after the interface declaration to adhere to the checklist.